packages/Avatar/src/utils.ts

```diff
@@ -1,9 +1,21 @@
 import { Theme } from '@xstyled/styled-components'
+import { ThemeSecondaryColors } from '@welcome-ui/core'
+
+const subColorByNumber: {
```

Comment:
You can manage to avoid naming colors here for `getSeededColor` can work without it I think.